source/framework/sensitivity/inc/TRestLikelihood.h

```diff
@@ -0,0 +1,71 @@
+/*************************************************************************
+ * This file is part of the REST software framework.                     *
+ *                                                                       *
+ * Copyright (C) 2016 GIFNA/TREX (University of Zaragoza)                *
+ * For more information see https://gifna.unizar.es/trex                 *
+ *                                                                       *
+ * REST is free software: you can redistribute it and/or modify          *
+ * it under the terms of the GNU General Public License as published by  *
+ * the Free Software Foundation, either version 3 of the License, or     *
+ * (at your option) any later version.                                   *
+ *                                                                       *
+ * REST is distributed in the hope that it will be useful,               *
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of        *
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the          *
+ * GNU General Public License for more details.                          *
+ *                                                                       *
+ * You should have a copy of the GNU General Public License along with   *
+ * REST in $REST_PATH/LICENSE.                                           *
+ * If not, see https://www.gnu.org/licenses/.                            *
+ * For the list of contributors see $REST_PATH/CREDITS.                  *
+ *************************************************************************/
+
+#ifndef REST_TRestLikelihood
+#define REST_TRestLikelihood
+
+#include "TRestMetadata.h"
+
+/// UPDATE Write here a brief description. Just one line!
+class TRestLikelihood : public TRestMetadata {
```

Comment:
Thi is a new class? I think shouldn't be added under this PR

Reply:
Right, this ended here by error! There is a new draft PR dedicated to that here https://github.com/rest-for-physics/framework/pull/355

Reply:
Removed here 6a3055627424c8cf01eca47ca3144f7a50425650